css/styles.css

```diff
@@ -271,6 +271,14 @@ body {
   to {opacity: 1}
 }
 
+.hidden {
+  display: none;
+}
+
+.shown {
```

Comment:
@mikemorris I'll tinker with this and see what I can do. They are block level elements by default and I remember thinking the same thing only to find that the quotes weren't appearing at all without explicitly redefining the display property to block.